Tests/Functional/SeleniumSessionTest.php

```diff
@@ -6,11 +6,11 @@
 
 /**
  * @group selenium
- */
+ *
```

Comment:
No. I was  a try to restore tests and selenium1 will be removed from the bundle in the next version.